pisa/stages/data/sqlite_loader.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import absolute_import, print_function, division
+
+import numpy as np
+import pandas as pd
+import sqlite3
+from pisa import FTYPE
+from pisa.core.stage import Stage
+from pisa.utils import vectorizer
+from pisa.utils.resources import find_resource
+from pisa.utils.profiler import profile
+from pisa.core.container import Container
+
+
+class sqlite_loader(Stage):
+    """
+    SQLite loader PISA Pi class
+    Parameters
+    ----------
+    database : path to sqlite database
+    **kwargs
+        Passed to Stage
+    """
+    def __init__(
+        self,
+        database,
+        output_names,
+        add_these =  ['energy', 'zenith'],
+        is_retro = False,
```

Comment:
could we here simply specify `_postfix` to make the stage able to load any reco.....for example also FLERCNN etc from the sqlite database?

Reply:
In principle but I would prefer not to. This dataloader is compatible only with the sqlite databases from graphnet, and we do not extract flercnn and there is not a plan to do so in the future.

Reply:
Sorry, "FLERCNN" is just an example. What I'm trying to say is that we could make the code more general / flexible. Instead of a boolean flag that allows for exactly two options and is fixed on the idea of comparing to "retro", we could simple here pass "retro" or "pred" or anything else like "banana" or "sharks_with_lasers" as a string. Then you can decide inside the function what happens in each case. Even assert that it should be ` in ('pred', 'retro')` right now if you want.

It's a very small detail, but I like to find ways to make stuff more general and future proof whenever possible.